Makefile

```diff
@@ -10,7 +10,30 @@ DATE := $(shell date +%Y-%m-%d)
 export DATE
 
 # CONTAINER_RUNTIME defines the container runtime to use for building the bundle image.
-CONTAINER_RUNTIME := $(shell command -v podman 2> /dev/null || echo docker)
+.PHONY: check-container-runtime
+check-container-runtime:
+	@if podman ps >/dev/null 2>&1; then \
+		echo ""; \
+		echo "============================================"; \
+		echo " ✅ Using Podman the container runtime"; \
+		echo "============================================"; \
+		CONTAINER_RUNTIME=podman; \
+	elif docker ps >/dev/null 2>&1; then \
+		echo ""; \
+		echo "============================================"; \
+		echo " ✅ Using Docker the container runtime"; \
+		echo "============================================"; \
+		CONTAINER_RUNTIME=docker; \
+	else \
+		echo ""; \
+		echo "============================================"; \
+		echo " ❌ Neither Podman nor Docker daemon is running"; \
+		echo "============================================"; \
+		exit 1; \
+	fi
+
+
+CONTAINER_RUNTIME := $(shell cat .container_runtime 2>/dev/null || echo docker)
```

Comment:
```suggestion
```

Reply:
Please check this new commit d7039703c3e52a871d3fe1ae2aa9861c648bc012.  **CONTAINER_RUNTIME** is used across multiple makefile commands.  We are only doing a additional check for the local development. So, that the user don't have to change anything manually in the makefile if the user wants to use Docker daemon instead of Podman.